spec.html

```diff
@@ -4627,7 +4628,7 @@ <h1>ToBoolean ( _argument_ )</h1>
               Number
             </td>
             <td>
-              If _argument_ is *+0*, *-0*, or *NaN*, return *false*; otherwise return *true*.
+              If _argument_ is *+0*<sub>𝔽</sub>, *-0<sub>𝔽</sub>*<sub>𝔽</sub>, or *NaN*, return *false*; otherwise return *true*.
```

Comment:
You have an error here.

```suggestion
              If _argument_ is *+0*<sub>𝔽</sub>, *-0<sub>𝔽</sub>*, or *NaN*, return *false*; otherwise return *true*.
```

But I still think `+0` and `-0` can be used without the subscript. We can also use "is zero" as shorthand for this.

Reply:
> We can also use "is zero" as shorthand for this.

I would prefer we not. "is zero" is ambiguous where "is +0 or -0" is not.

Reply:
Ambiguous how? If we think it's ambiguous, we can clarify it in the notational conventions section. I think it'll be used in a good number of places.

Reply:
Ambiguous because it is not clear if it covers both. We can give it a definition, yes, but I'd prefer we just use the unambiguous phrasing which does not require a definition.

We [talked about this on the editor call](https://github.com/tc39/ecma262/pull/1963#issuecomment-624915202) a few weeks ago; I thought you agreed with this position then.

Reply:
I thought that decision was just about using "is" vs "=".